src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTaskTest.java

```diff
@@ -52,16 +54,19 @@ public void setUp() {
     public void should_mark_as_deleted_complete_envelope() throws Exception {
         // given
         final String containerName1 = "container1";
+        final String leaseId = "LEASEID";
```

Comment:
Maybe we can capture this lease id so to verify it was actually used when deleting?

-- edit --

oh this is integration testing. erm. capturing can be moved to unit test then ;)